src/pages/services/index.jsx

```diff
@@ -1,9 +1,62 @@
+import styles from "@/styles/pages/services/services.module.scss";
+import ImageSlider from '@/components/services/imageSlider.jsx';
 
-export default function Services()
-{
-    return(
-        <>
-        Services yay
-        </>
+export default function Services() {
+    return (
+        <div className={styles.mainContainer}>
+
+            <div className={styles.servicesContainer}>
+                <div>
+                    <h1 className={styles.pageHeading}>Our Services</h1>
+                </div>
+
+                <div className={styles.boxContainer}>
+
+                    <div className={`${styles.serviceBox} ${styles.topServiceBox}`}>
+
+                        <h3 className={styles.boxHeadingTop}> Primary Care Services </h3>
```

Comment:
Just a small tag change here to `h2`. Best practice is to always use headings in the right order. So, `h2` should always be used before you use `h3` and so on.